app/models/person.rb

```diff
@@ -70,21 +71,8 @@ def self.parse_message(message: nil)
       orcid: orcid,
       given_name: given_name,
       family_name: family_name,
-      other_names: other_names,
```

Comment:
Does this change would `break` the public interface for someone that has integrated with `other_names` already? if it does then i would think this will need to be documented or managed so other integrations do not `fail`. That probably will dictate wether this change is a 'MINOR' or a 'MAJOR' release